pkg/asset/store/data

```diff
@@ -0,0 +1 @@
+../../../data/data
```

Comment:
what is this file?

Reply:
This is a link to that directory. `TestCreatedAssetsAreNotDirty` needs it to be able to generate the manifest templates.